app/Models/Multiplayer/PlaylistItemUserHighScore.php

```diff
@@ -130,4 +131,9 @@ public function updateWithScoreLink(ScoreLink $scoreLink): bool
             'total_score' => $score->total_score,
         ])->save();
     }
+
+    public function user(): BelongsTo
```

Comment:
oh right, this should be in relation function area?

Reply:
are we still doing that 🤔

Reply:
I think so for scopes, relations and laravel model setters/getters? 🤔